framework/src/modules/interoperability/base_interoperability_internal_methods.ts

```diff
@@ -466,7 +467,7 @@ export abstract class BaseInteroperabilityInternalMethod extends BaseInternalMet
 		const isReceivingChainLive = await this.isLive(
 			context,
 			receivingChainID,
-			timestamp ?? Date.now(),
+			timestamp ?? getCurrentTimestamp(),
```

Comment:
in the module, it should never use the machine timestamp.
Do we need this input?

Reply:
ah got it, I think when `interop.send()` is called from token module, it should pass `header.timestamp` instead of undefined. So on mainchain, in general, all the modules should pass timestamp as mandatory option when calling `interop.send()` method

Reply:
Related commit https://github.com/LiskHQ/lisk-sdk/pull/8271/commits/6e468eda8c619d019e261886558d483a13321143